acceptance/config/nodes/win2012r2-rubyx64.yaml

```diff
@@ -5,6 +5,15 @@ HOSTS:
     platform: el-7-x86_64
     hypervisor: vcloud
     template: redhat-7-x86_64
+    systembindir: /opt/puppetlabs/agent/bin
+    puppetconfdir: /etc/puppetlabs/agent
+    puppetcodedir: /etc/puppetlabs/agent
+    puppetvardir: /opt/puppetlabs/agent/cache
+    distmoduledir: /etc/puppetlabs/agent/modules
+    # puppet-agent 0.3.1 package on the master still uses /usr/share
+    sitemoduledir: /usr/share/puppet/modules
+    hieraconf: /etc/puppetlabs/puppet/hiera.yaml
+    puppetserver-confdir: /etc/puppetserver/conf.d/
```

Comment:
@Iristyle please update with the latest version from https://github.com/puppetlabs/puppet/commit/66c2e0605c1504ddc632d379a45440548df411d5